_gsocproposals/2020/proposal_ATLASMLcompression.md

```diff
@@ -0,0 +1,62 @@
+---
+title: Deep autoencoders for ATLAS data compression
+layout: gsoc_proposal
+project: ATLAS
+year: 2020
+organization: CERN
+---
+
+## Short description of the project 
+
+Storage is one of the main limiting factors to the recording of information from proton-proton collision events at the Large Hadron Collider, at CERN in Geneva. Hence, ATLAS uses a so-called ”trigger system”, which selects and sends interesting events to the data storage system while throwing away the rest. 
+
+However, if interesting events are buried in very large backgrounds and difficult to identify as signal by the trigger system, they will also be discarded together with the background. To alleviate this problem, we plan to reduce the size of the data that is recorded, and study compression algorithms that can be used directly within the trigger system. A variety of compression algorithms is already in use in high energy physics (see e.g. Ref. [1]). 
```

Comment:
Rather use: "see e.g. this paper" with direct link on "paper". Leave the reference as it is in the "Links" section, but remove the [1] (and maybe also the authors)

Reply:
Fixed, but left the authors and removed the link (so that only software links appear).